pkg/internal/controller/controller.go

```diff
@@ -115,14 +115,14 @@ func (c *Controller) Watch(src source.Source, evthdler handler.EventHandler, prc
 	c.watches = append(c.watches, watchDescription{src: src, handler: evthdler, predicates: prct})
 	if c.Started {
 		c.Log.Info("Starting EventSource", "source", src)
-		return src.Start(evthdler, c.Queue, prct...)
+		return src.Start(context.TODO(), evthdler, c.Queue, prct...)
```

Comment:
Should we modify `Watch` to also accept a context, or save it as part of `Start` and pass it in here?

Reply:
Propagated the context with the one passed into Start, given that seems the most consistent approach